src/get-started/java-binding.md

```diff
@@ -225,9 +277,9 @@ rules – see the corresponding section of our [documentation][transactions].
 Transactions are transmitted by external service clients to the framework as
 [Exonum messages][transactions-messages].
 A transaction message contains a header with the identifying information,
-such as an ID of the service this transaction belongs to and a transaction ID
-within that service; a payload containing transaction parameters;
-a public key of the author and a signature that authenticates them.
+such as a numeric ID of the service this transaction belongs to 
+and a transaction ID within that service; a payload containing transaction 
```

Comment:
> transaction ID

This identifier is called method ID in the core. Do you think "transaction ID" is better because transactions are not (yet) methods in EJB?

Reply:
We may use the newer term in this documentation, but in all the interfaces in upcoming 0.9 it is still named transaction id (incl. their Javadocs).